lib/writer/block-write-stream.js

```diff
@@ -56,12 +58,14 @@ class BlockWriteStream extends stream.Writable {
     this.flags = options.flags
     this.mode = options.mode
     this.autoClose = options.autoClose
+    this.maxRetries = 5
 
     this.position = 0
     this.bytesRead = 0
     this.blocksRead = 0
     this.bytesWritten = 0
     this.blocksWritten = 0
+    this.retries = 0
```

Comment:
Not that it really matters, but in other code within Etcher (I can't remember where now) the 'retry' was handled via an extra function parameter (which defaulted to 0), e.g.
```javascript
somefunc(params, tries=0) {
    doSomething()
    if (error)
        if (tries < maxRetries) {
            somefunc(params, tries + 1)
        } else {
            displayError()
        }
    }
}
```
whereas here you're storing the retry count on `this`.
Are there any pros or cons to either approach?